examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
```

Comment:
1. the first argument is a 3 Tuple. Therefore, (a, b, c) == a*frame.x + b*frame.y + c*frame.z

2. force is stored in a list thus, add_force was needed. It appends to the force_list.